balancer-js/src/modules/relayer/relayer.module.ts

```diff
@@ -147,8 +148,70 @@ export class Relayer {
     ]);
   }
 
-  static encodeUnwrapERC4626(params: EncodeUnwrapERC4626Input): string {
-    return relayerLibrary.encodeFunctionData('unwrapERC4626', [
+  static encodeUnwrapWstETH(params: EncodeUnwrapWstETHInput): string {
+    return relayerLibrary.encodeFunctionData('unwrapWstETH', [
+      params.sender,
+      params.recipient,
+      params.amount,
+      params.outputReference,
+    ]);
+  }
+
+  static encodeUnwrap(
+    params: EncodeUnwrapInput,
+    linearPoolType: string
+  ): string {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let unwrapType: any;
+
+    /**
+     * Other unwrap types available on BatchRelayerLibrary that does not seem to
+     * have a respective Linear pool type in the SDK:
+     * - unwrapCompoundV2
+     * - unwrapShareToken
+     * - unwrapUnbuttonToken
+     * - unwrapWstETH
```

Comment:
So it seems like these pool types don't currently exist in SG but do exist in Relayer library. So we can't map them for now.